metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceImpl.java

```diff
@@ -156,12 +204,20 @@ public void notifyOfPartitionDeletion(final MetacatDeleteTablePartitionPostEvent
         final String name = event.getName().toString();
         final long timestamp = event.getRequestContext().getTimestamp();
         final String requestId = event.getRequestContext().getId();
+        final TablePartitionsUpdatePayload partitionsUpdatePayload;
+        if (this.config.isSnsNotificationAttachPartitionIdsEnabled() && event.getPartitions() != null) {
+            partitionsUpdatePayload = createTablePartitionsUpdatePayload(event.getPartitions(), event);
```

Comment:
Isn't this wrong? This needs to go in as deleted partitions list?

Reply:
Ouch. this is not right.